src/main/java/seedu/address/commons/events/ui/DeselectionEvent.java

```diff
@@ -5,7 +5,7 @@
 /**
  * Represents the current list becoming empty.
  */
-public class EmptyListEvent extends BaseEvent {
+public class DeselectionEvent extends BaseEvent {
```

Comment:
will there be any implications of removing the `EmptyListEvent`?

Reply:
I renamed the entire thing to `DeselectionEvent` so everything that used `EmptyListEvent` before will use `DeselectionEvent` now to achieve the same effect

Reply:
nice